scripts/gen.py

```diff
@@ -96,25 +97,37 @@ def write_file(filename, contents):
             workflow_specs_json = serialize_workflow_specs_for_process_id(process_id, ctx)
             write_file(filename, workflow_specs_json)
             ctx.process_id_by_json_file[filename] = process_id
+            ctx.json_file_by_process_id[process_id] = filename
 
 def generate_manifest(ctx):
     manifest = {
         "bpmn_file_by_process_id": ctx.bpmn_file_by_process_id,
         "called_element_ids_by_process_id": ctx.called_element_ids_by_process_id,
         "process_id_by_workflow_spec_json": ctx.process_id_by_json_file,
         "process_ids_by_bpmn_file": ctx.process_ids_by_bpmn_file,
+        "workflow_spec_json_by_process_id": ctx.json_file_by_process_id,
     }
 
     with open("manifest.json", "w") as f:
         f.write(json.dumps(manifest, sort_keys=True, indent=2))
```

Comment:
Since this is script that is just run to generate the spec files which are then used by other SpiffWorkflow based tooling, I'm ok with just letting the exceptions bubble up and dump to the terminal for easier debugging.